internal/generationtest/generation_test.go

```diff
@@ -15,6 +16,7 @@ var (
 		" By default, will use files named BUILD.in as the BUILD files before running gazelle.")
 	buildOutSuffix = flag.String("build_out_suffix", ".out", "The suffix on the expected BUILD.bazel files after running gazelle. Defaults to .out. "+
 		" By default, will use files named BUILD.out as the expected results of the gazelle run.")
+	timeoutSeconds = flag.Int("timeout_seconds", 2, "Number of seconds to allow the gazelle process to run before killing.")
```

Comment:
Could we use the built in [duration flag](https://pkg.go.dev/flag#Duration) instead of passing untyped integers? That will allow specifying `2s` or `5m` as valid values, which are self-unit-documenting.

Reply:
Done, but left the bazel interface as still being int seconds - I can switch that to be a string if you'd prefer